config.toml

```diff
@@ -1,8 +1,8 @@
 # The URL the site will be built for
 base_url = "https://accio-demo.netlify.app/"
 
-title = "🧙‍♂️ Vijard"
-description = "Welcome to my world of leisure"
+title = "Accio"
+description = "Spell Bound theme for Zola SSG"
```

Comment:
"A" Spell Bound theme for Zola SSG

Reply:
Would add the "A", for the Resume part i guess it's ok to have it , as the user needs to make his own config.toml where the links for the header needs to be mentioned, will add an if in config.toml for config.toml whose true value would add the Resume part to the header

Reply:
That's much better